src/grpc_requests/aio.py

```diff
@@ -556,13 +592,27 @@ async def register_service(self, service_name):
             logger.debug(f"{service_name} registration complete")
         await super(ReflectionAsyncClient, self).register_service(service_name)
 
+    async def refresh_service_registrations(self):
+        """
+        Clears the existing registered services and freshly re-registers them.
+        Useful if the gRPC server adds or removes services dynamically.
+
+        Regular re-registration helps ensure the client's service registry remains
+        synchronized with the current state of the server, thus accommodating
+        service changes that occur during the lifespan of the connection.
+        """
+        self._service_methods_meta.clear()
```

Comment:
Would you need to also clear and re-register file descriptors, or clean up the descriptor pool in general for a proper refresh?